File: AtividadesEtec2019/TPA/Lista3Laco-master/Em Python/ExercicioD.py
# -*- coding: utf-8 -*-
km = input("Digite a distância da estrada em km: ")
km = float(km)
mes = 3.75
while mes <= km:
    d = mes
    mes = mes + 3.75
d = d / 3.75
d = d + 1.0
d = str(d)
print("Vai demorar "+d+" meses para terminar a estrada")

File: AtividadesEtec2019/TPA/2-Bimestre/Exercicios lista 3/ExercicioD.java
import java.util.Scanner;
public class ExercicioD {
	public static void main(String[]Args) {
		Scanner ler = new Scanner(System.in);
		double km,mes,d = 0,s = 0; //km = distancia da estrada, mes = meses, d = calculo do mes
		System.out.println("Digite a distancia da estrada em km");
		km =ler.nextInt();
		mes=3.75;
		while (mes<=km) {
			d=mes;
			mes=mes+3.75;
		}d=d/3.75;
		d=d+1.0;
		System.out.println("Vai demorar "+d+" meses para terminar a estrada");
		
	}

}